### PR TITLE
trace: Move FW description log location

### PR DESCRIPTION
**Pull request by ktrzcinx**
This log should be added just after buffer initialization,
to make it the first one in the output file - for easy identyfication.

Signed-off-by: Karol Trzcinski <karolx.trzcinski@linux.intel.com>

### CODE REVIEW

File: src/trace/dma-trace.c [hunk 1]
```diff
@@ -328,6 +332,9 @@ int dma_trace_enable(struct dma_trace_data *d)
 		goto out;
 #endif
 
+	/* flush fw description message */
+	trace_flush();
+
```
Comment (dbaluta) [inline src/trace/dma-trace.c:337]: @ktrzcinx Why, with this change, tr_info and trace_flush() are no longer close one to each other?

Comment (ktrzcinx) [reply src/trace/dma-trace.c:337]: To flush after `dma_trace_start` - possible part of trace initialization while writing trace log just after buffer initialization (after moving it one line below then it's not the first trace log)

Comment (dbaluta) [reply src/trace/dma-trace.c:337]: Ok.

**Review by dbaluta — APPROVED** [review]

**Review by lgirdwood — COMMENTED** [review]
Can we make this tr_err() so that this message also hits the error log ? The inline comment can explain why it's using tr_err().

**Comment by ktrzcinx** [comment]
Are we sure changing it to `tr_err()`  won't break CI ?

**Comment by lgirdwood** [comment]
@ktrzcinx it may do, but CI folks are asking for a check for erro log. @zrombel @xiulipan @aiChaoSONG any commens, this PR could add an initial "hello world" message in th eerror log to show CI it's working (message is NOT an error).

**Comment by zrombel** [comment]
Not an issue for Quickbuild CI.

**Comment by xiulipan** [comment]
@ktrzcinx It would be OK for Jenkins CI. We are trying to modify our test scripts to check a "hello world" message to make sure trace function is working.

**Comment by lgirdwood** [comment]
SOFCI TEST

**Comment by lgirdwood** [comment]
Known Jenkins CI issues.

**Pull request merged**